Makefile

```
@@ -277,24 +289,40 @@ bootstrap: configs/flaresolverr/config/chromedriver
+# `backup` is an alias for `backup-configs`: the lean archive is the right
+# default for routine backups. Both restore through the same `restore-configs`
+# target (`restore-full` is just an alias for it too); a full backup is only
+# worth the extra size if you actually need the things it keeps that the lean
+# one strips: Jellyfin's metadata/cache and Recyclarr's cloned git repos.
+# *arr MediaCover artwork and scheduled backups are no longer among them: both
+# now live under data/ (MEDIA_COVERS_FOLDER and APP_BACKUPS_FOLDER), so neither
+# mode archives them. The excludes below are kept for setups predating that
+# move. See docs/STORAGE.md.
 backup: backup-configs
 
 backup-configs:
```

checkmake (0.3.2)

[warning] Target body for "backup-configs" exceeds allowed length of 5 lines (10).

(maxbodylength)